zerver/views/users.py

```diff
@@ -457,7 +460,7 @@ def add_bot_backend(
         if service_name not in [bot.name for bot in EMBEDDED_BOTS]:
             raise JsonableError(_("Invalid embedded bot name."))
 
-    if not form.is_valid():
+    if not form.is_valid():  # nocoverage
```

Comment:
This might be unreachable now.

Reply:
I think it's still reachable with a 200 character name, but I'm not sure, and we might eliminate that possibility with additional `REQ` parameters instead; added a comment next to this.

Reply:
It’s not; we already checked for long full names in `check_full_name`, and there’s no maximum on the email.